docs/tutorial/first-app.md

```diff
@@ -104,6 +113,14 @@ RewriteCond %{REQUEST_FILENAME} !-f
 RewriteRule . index.php [L]
 ```
 
+nginx does not use `.htaccess` files, so you will need to add the following to your server configuration in the `location` block:
+
+```
+if (!-e $request_filename){
+    rewrite ^(.*)$ /index.php break;
```

Comment:
Can you add a comment about how `index.php` is actually whatever entry point into your application... I know some people don't use index.php instead use something like api.php